CMakeLists.txt

```diff
@@ -5,6 +5,12 @@ find_program(GIT_PROGRAM git)
 find_program(CMAKE_FORMAT_PROGRAM cmake-format)
 find_package(Python)
 
+if(NOT CMAKE_FORMAT_PROGRAM AND Python_FOUND)
+  message(STATUS "Format.cmake: cmake-format not found, installing cmakelang")
+  execute_process(COMMAND ${Python_EXECUTABLE} -m pip install cmakelang)
```

Comment:
Not hardcoding the version can lead to issues where different systems will run the formatter differently, as every user has the most recent version of the time that the configure script was first run.